deploy/static/cert-manager-openshift-routes.yaml

```diff
@@ -1,130 +1,144 @@
+    kind: ClusterRoleBinding
+    metadata:
+      name: cert-manager-openshift-routes
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: cert-manager-openshift-routes
+    subjects:
+    - kind: ServiceAccount
+      name: cert-manager-openshift-routes
+      namespace: ${CERT_MANAGER_NAMESPACE}
+
+  - apiVersion: apps/v1
+    kind: Deployment
     metadata:
+      name: cert-manager-openshift-routes
+      namespace: ${CERT_MANAGER_NAMESPACE}
       labels:
         app.kubernetes.io/name: cert-manager-openshift-routes
         app.kubernetes.io/version: "$RELEASED_VERSION"
         app.kubernetes.io/component: controller
-        app.kubernetes.io/part-of: cert-manager
+        app.kubernetes.io/part-of: ${CERT_MANAGER_NAMESPACE}
     spec:
-      serviceAccountName: cert-manager-openshift-routes
-      automountServiceAccountToken: true
-      containers:
-        - name: cert-manager-openshift-routes
-          image: "ghcr.io/cert-manager/cert-manager-openshift-routes:$RELEASED_VERSION"
-          args:
-          - -v=5
-          ports:
-          - containerPort: 6060
-            name: readiness
-            protocol: TCP
-          - containerPort: 9042
-            name: metrics
-            protocol: TCP
-          readinessProbe:
-            httpGet:
-              port: readiness
-              path: "/readyz"
-            initialDelaySeconds: 3
-            periodSeconds: 5
-            timeoutSeconds: 3
+      replicas: 1
+      selector:
+        matchLabels:
+          app.kubernetes.io/name: cert-manager-openshift-routes
+          app.kubernetes.io/version: "$RELEASED_VERSION"
+          app.kubernetes.io/component: controller
+          app.kubernetes.io/part-of: ${CERT_MANAGER_NAMESPACE}
+      template:
+        metadata:
+          labels:
+            app.kubernetes.io/name: cert-manager-openshift-routes
+            app.kubernetes.io/version: "$RELEASED_VERSION"
+            app.kubernetes.io/component: controller
+            app.kubernetes.io/part-of: ${CERT_MANAGER_NAMESPACE}
+        spec:
+          serviceAccountName: cert-manager-openshift-routes
+          automountServiceAccountToken: true
+          containers:
+            - name: cert-manager-openshift-routes
+              image: "ghcr.io/cert-manager/cert-manager-openshift-routes:$RELEASED_VERSION"
+              args:
+              - -v=5
+              ports:
+              - containerPort: 6060
+                name: readiness
+                protocol: TCP
+              - containerPort: 9042
+                name: metrics
+                protocol: TCP
+              readinessProbe:
+                httpGet:
+                  port: readiness
+                  path: "/readyz"
+                initialDelaySeconds: 3
+                periodSeconds: 5
+                timeoutSeconds: 3
+              env:
+                - name: CERT_MANAGER_NAMESPACE
+                  value: ${CERT_MANAGER_NAMESPACE}
```

Comment:
What is this environment variable for? It doesn't seem to be used by the controller anywhere: https://github.com/search?q=repo%3Acert-manager%2Fopenshift-routes%20CERT_MANAGER_NAMESPACE&type=code